dataproc/README.md

```diff
@@ -8,7 +8,7 @@ demonstrated here could also be accomplished using the Cloud Console or the gclo
 `list_clusters.py` is a simple command-line program to demonstrate connecting to the
```

Comment:
Meta: This readme is enormous. Is there a documentation page that it could link to instead?

Reply:
We do: https://cloud.google.com/dataproc/docs/tutorials/python-library-example . So yeah fair point, maybe I should cut this whole README, make sure all info's on that page?

Reply:
That would be strongly preferable.

Reply:
Ok, I pared down the README. I basically left all the information to actually run the samples, killed all the associated info and link to the tutorial at the top. I'll make a docs CL to add the note about reading from GCS with PySpark. LMK if that makes sense.